queries/reports/filters.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable prettier/prettier */
+import { sparqlEscapeDate, sparqlEscapeUri } from 'mu'; // eslint-disable-line
+import { sparqlEscapeDateLocal } from '../utils.js';
+
```

Comment:
Just for understandability of the code, I'd recommend adding a short comment here to clarify that these functions do not generate complete SPARQL queries, but rather parts of the query generated in `queries/reports/index.js`